src/app/shared/location-riddle-post/location-riddle-post.component.html

```diff
@@ -0,0 +1,100 @@
+<ion-card style="width: 90vw; min-height: 37rem; max-width: 40rem" color="light">
+	<ion-card-header class="ion-padding">
+		<ion-card-title>
+			<ion-item lines="none">
+				<ion-avatar aria-hidden="true" slot="start">
+					<ion-img src="https://ionicframework.com/docs/img/demos/avatar.svg"></ion-img>
+				</ion-avatar>
+				<div style="display: flex; justify-content: space-between; flex-grow: 1">
+					<ion-label color="dark">
+						<h2>{{ locationRiddle().username }}</h2>
+						<p>{{ locationRiddle().createdAt | date: 'short' }}</p>
+					</ion-label>
+					<app-rating
+						[rating]="locationRiddle().rating"
+						(makeRating)="rateLocationRiddle.emit($event)"
+						[ratingError]="ratingError"
+					></app-rating>
+				</div>
+			</ion-item>
+		</ion-card-title>
+	</ion-card-header>
+	<ion-card-content style="height: 40vh; min-height: 30rem">
+		@if (showMap()) {
+			<app-map
+				(placeMarker)="placeGuess($event)"
+				[marker]="marker()"
+				[guesses]="guesses()"
+				[userGuess]="userGuess()"
+				[solution]="solution()"
+				[solved]="locationRiddle().solved"
+				[zoom]="mapZoom()"
+				[center]="mapCenter()"
+				(zoomChange)="mapZoom.set($event)"
+				(centerChange)="mapCenter.set($event)"
+			></app-map>
+		} @else {
+			<ion-img src="data:image;base64, {{ locationRiddle().locationRiddleImage }}"></ion-img>
+			<ion-item
+				id="open-modal-{{ locationRiddle().locationRiddleId }}"
+				expand="block"
+				class="ion-no-padding"
+				lines="none"
+				style="margin-top: 15px; margin-bottom: 5px"
+			>
+				<ion-icon size="small" name="chatbox-outline"></ion-icon>
+				<p style="padding-left: 0.3rem">add a comment...</p>
+			</ion-item>
+		}
+		<ion-fab horizontal="start" slot="fixed" vertical="bottom">
+			<ion-fab-button (click)="toggleMap()">
+				<ion-icon name="{{ showMap() ? 'image' : 'map' }}-outline"></ion-icon>
+			</ion-fab-button>
+		</ion-fab>
```

Comment:
I think this was me but we should move the comment outside of the else:
```suggestion
		} @else {
			<ion-img src="data:image;base64, {{ locationRiddle().locationRiddleImage }}"></ion-img>
		}
		<ion-fab horizontal="start" slot="fixed" vertical="bottom">
			<ion-fab-button (click)="toggleMap()">
				<ion-icon name="{{ showMap() ? 'image' : 'map' }}-outline"></ion-icon>
			</ion-fab-button>
		</ion-fab>
		<ion-item
                    id="open-modal-{{ locationRiddle().locationRiddleId }}"
                    expand="block"
                    class="ion-no-padding"
                    lines="none"
                    style="margin-top: 15px; margin-bottom: 5px"
                    >
                    <ion-icon size="small" name="chatbox-outline"></ion-icon>
                    <p style="padding-left: 0.3rem">add a comment...</p>
		</ion-item>
```

Reply:
fix: https://github.com/uzh-ase-fs24/frontend/pull/45/commits/ac7788ed82aa939a9792e5d5a62547e6d4751e74

Reply:
fix: https://github.com/uzh-ase-fs24/frontend/pull/45/commits/ac7788ed82aa939a9792e5d5a62547e6d4751e74